current/runtime/fees.md

```diff
@@ -4,97 +4,32 @@ lang: en
+execute an extrinsic can vary, Substrate provides a flexible mechanism called "weights" to
+characterize the _time_ it takes to execute an extrinsic.
 
-The fee system is heavily linked to the [weight system](/current/learn-substrate/weight.md). Make 
-sure to understand weights before reading this document.
-
-## Default Fees
-
-The default fee system uses the uses _fixed_ weights represented by the following enum:
-
-```rust
-pub enum SimpleDispatchInfo {
-    /// A normal dispatch with fixed weight.
-    FixedNormal(Weight),
-    /// A normal dispatch with the maximum weight.
-    MaxNormal,
-    /// A normal dispatch with no weight.
-    FreeNormal,
-    /// An operational dispatch with fixed weight.
-    FixedOperational(Weight),
-    /// An operational dispatch with the maximum weight.
-    MaxOperational,
-    /// An operational dispatch with no weight.
-    FreeOperational,
-}
-```
-
-This enum groups all dispatches into _normal_ or _operational_ and gives them a fixed weight. This
-means that the arguments of a dispatchable function do not affect the weight. A dispatch classified
-as _operational_ is exempt from paying both the `base_fee` and the `length_fee`.
-
-### Free Normal
-
-This means that this function has no weight. It will not contribute to block fullness at all, and no
-weight-fee is applied.
-
-```rust
-#[weight = SimpleDispatchInfo::FreeNormal]
-pub fn some_normal_function_light() { noop(); }
-```
-
-### Fixed Normal
-
-This function will always have a weight `10`.
-
-```rust
-#[weight = SimpleDispatchInfo::FixedNormal(10)]
-pub fn some_normal_function_heavy() { some_computation(); }
-```
-
-### Fixed Operational
-
-This function will have a fixed weight but can consume the reserved operational portion as well.
-
-```rust
-#[weight = SimpleDispatchInfo::FixedOperational(20)]
-pub fn mission_critical_function() { some_sudo_op(); }
-```
-
-### Default
-
-This function will automatically get `#[weight = SimpleDispatchInfo::default()]`.
-
-```rust
-pub fn something_else() { noop(); }
-```
-
-> **Note:** Be careful! The default implementation of `SimpleDispatchInfo` resolves to
-> `FixedNormal(10_000)`. This is due to how things work in `substrate-node` and the desired
-> granularity of substrate. Even if you want to use the `SimpleDispatchInfo`, it is very likely that
-> you would want it to have a different `Default`.
+Although an extrinsic's weight is only one component of the fee charged to its sender, it is
+recommended to understand the [weight system](/current/learn-substrate/weight.md) before reading
+this document.
 
 ## Fee Calculation
 
-The final fee of a dispatch is calculated using the weight of the function and a number of
-configurable parameters.
+The final fee of a dispatch is calculated using the weight of the dispatchable function and a number
+of configurable parameters.
 
 ### Inclusion Fee
 
-A transaction fee consists of three parts:
+An extrinsic fee consists of three parts:
```

Comment:
Actually here we can be more specific. Inherents don't have fees because they are not signed.
```suggestion
A transaction fee consists of three parts:
```